test/distributed_exec.jl

```diff
@@ -1498,6 +1498,14 @@ if true
 end
 @test x == map(_->sin(2), 1:2)
 
+try
+    remotecall_fetch(sqrt, 2, -1)
```

Comment:
should ensure some error gets thrown in the try here, so it doesn't silently skip the catch if it ever gets broken in a way that it doesn't